src/config/config.ts

```diff
@@ -9,25 +9,25 @@ const coreBackApiUrl = appEnv.isLocal
   : appEnv.getServiceURL("core-back-api");
 
 export default {
-  ANDROID_APP_ID: process.env.ANDROID_APP_ID || "uk.gov.documentchecking",
+  ANDROID_APP_ID: process.env.ANDROID_APP_ID ?? "uk.gov.documentchecking",
```

Comment:
I think we potentially care - but in general empty string might be a valid value!

We could rewrite this to validate config a bit more effectively, like we do for the api test config (for example), but trying to avoid too much refactoring at once...

Reply:
(I changed it because Sonar was flagging it as an issue)